### DISCUSSION TITLE
[Craft 5] Palette icon overlaps new toggle sidebar UI

### DISCUSSION THREAD

**Discussion by kylecotter** [trendyminds/craft-palette#35] [Q&A] [answered]
Craft 5 adds a new toggle sidebar button in the bottom left of the control panel which overlaps with the Palette icon placement.

Bottom right is where the debug toolbar in its collapsed state lives. Not sure where else the Palette icon could live in the CP, but I'd assume either bottom right or higher up on the left.

**Comment by aaronbushnell** [comment]
Thanks for logging this, @kylecotter! Yeah, I noticed that too. I'll probably wait until beta 2 before making any dramatic changes _juuuuust_ in the off chance that the sidebar is still a work in progress. But I'll make sure to knock this one out before the RC goes out!

**Comment by aaronbushnell** [comment, marked as answer]
4.0 is out now with a lot of big changes! I no longer show the Palette icon which means this will no longer have any UI conflicts with Craft 5's UI changes!

**Comment by wmdhosting** [comment]
Ho to open Pallete on windows 11 now.. ?  tryed all keys combinations but nothing worked

**Reply by aaronbushnell** [reply]
Have you tried Ctrl+K? The `meta` option should handle both CMD (macOS) and CTRL (Windows)

**Comment by wmdhosting** [comment]
yes.  CTRL+K  opens Google search in Chrome

**Comment by wmdhosting** [comment]
in Firefox similar thing
![Snag_30ac2bb](https://github.com/trendyminds/craft-palette/assets/76176291/4bcd84dd-095a-4588-9bb0-4e348dd1d5a4)

**Reply by aaronbushnell** [reply]
Have you tried updating to the latest version of Palette?

**Comment by wmdhosting** [comment]
yes. just couple minutes before  i posted here..

**Reply by aaronbushnell** [reply]
Just posted a fix in 4.0.4 for this!

**Comment by wmdhosting** [comment]
thx works now :)

**Comment by kylecotter** [comment]
@aaronbushnell awesome! possible to merge these into the Craft 5 branch or are you still refactoring for 5?

**Reply by aaronbushnell** [reply]
Hey @kylecotter, still doing a bit of work to get it ready for Craft 5! Hope to have something ASAP!

**Reply by aaronbushnell** [reply]
Craft 5 version of Palette is out!
